.genx/templates/web-components/component/component.template.hbs

```diff
@@ -0,0 +1,23 @@
+import { html, whenElse } from '@genesislcap/web-core';
+import { getViewUpdateRightComponent } from '../../../utils';
+import type { {{pascalCase tile.componentName}} } from './{{kebabCase tile.title}}';
+{{#if tile.config.createFormUiSchema}}
+import { uiSchemaCreate } from './{{kebabCase tile.title}}.create.form.schema';
+{{/if}}
+{{#if tile.config.uischema}}
+import { uiSchemaCreate } from './{{kebabCase tile.title}}.create.form.schema';
+{{/if}}
+{{#if tile.config.updateFormUiSchema}}
+import { uiSchemaUpdate } from './{{kebabCase tile.title}}.update.form.schema';
+{{/if}}
```

Comment:
We can call these exports a bit more human-friendly e.g. createFormSchema, updateFormSchema etc.

Reply:
[68a6087](https://github.com/genesiscommunitysuccess/blank-app-seed/pull/275/commits/68a6087374efa86b795123814f3f9e282506a562)